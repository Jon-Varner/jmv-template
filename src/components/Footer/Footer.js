import React from 'react';

const Footer = React.memo(props => (
  <footer>
    <ul>
      <li>&copy; copyright info</li>
    </ul>
    <ul>
      <li>Contact info</li>
      <li>More contact info</li>
    </ul>
  </footer>
));

export default Footer;
