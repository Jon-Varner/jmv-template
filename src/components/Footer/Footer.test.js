import React from 'react';
import { render, cleanup } from 'react-testing-library';
import 'jest-dom/extend-expect';

import Footer from './Footer';

afterEach(cleanup);

it('renders', () => {
  const { asFragment } = render(<Footer someProp="value" />);
  expect(asFragment()).toMatchSnapshot();
});
