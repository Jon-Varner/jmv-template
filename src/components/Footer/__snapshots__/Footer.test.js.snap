// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <footer>
    <ul>
      <li>
        Â© copyright info
      </li>
    </ul>
    <ul>
      <li>
        Contact info
      </li>
      <li>
        More contact info
      </li>
    </ul>
  </footer>
</DocumentFragment>
`;
